### PR TITLE
Clarifying help text

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: webapp/components/search_results.jsx [hunk 1]
```diff
@@ -141,14 +141,14 @@ export default class SearchResults extends React.Component {
                     <h4>
                         <FormattedMessage
```
Comment (asaadmahmoodspin) [inline webapp/components/search_results.jsx]: Add a <strong> tag tag surrounds the <FormattedMessage>.

Comment (it33) [inline webapp/components/search_results.jsx]: @asaadmahmoodspin don't understand your feedback, sorry?

Comment (enahum) [inline webapp/components/search_results.jsx]: What he's saying is that it should be like this

```javascript
<h4>
    <strong>
        <FormattedMessage
            id='search_results.noResults'
            defaultMessage='No results found. Try again?'
        />
    </strong>
</h4>
```

**Comment by enahum** [comment]
+1 after @asaadmahmoodspin suggestions

**Comment by asaadmahmoodspin** [comment]
+1 looks like we've increased the message content so I'm okay without the `<strong>` tags now, we can still go ahead and add it back again if we want.

**Comment by it33** [comment]
Thanks @enahum, discussed with @asaadmahmoodspin and we're going to see how it looks without bold for now--since that's an additional change, and we have time to tweak before 3.2 release. 

@asaadmahmoodspin thanks again for the ideas on this, a lot better than what I was originally thinking.

File: webapp/components/search_results.jsx [hunk 2]
```diff
@@ -141,14 +141,14 @@ export default class SearchResults extends React.Component {
...
                             id='search_results.noResults'
-                            defaultMessage='NO RESULTS'
+                            defaultMessage='No results found. Try again?'
```
Comment (crspeller) [inline webapp/components/search_results.jsx]: I think the `Try again?` is too verbose.

Comment (it33) [inline webapp/components/search_results.jsx]: Good topic to bring up, 

In my mind, we have "fast, obvious, forgiving" guidelines, we don't have a guidelines around being verbose, or how to make trade-offs between verbose-ness and other priorities. 

E.g. Our LDAP settings have a lot of help text that isn't needed by LDAP experts, but when we add it our support questions are significantly reduced. 

Propose we figure out a guideline so we can be consistent in the trade-off between helping the least savvy user and having more text than experts need. 

Until we have a standard, I think stay focused on "fast, obvious, forgiving." 

For this text change specifically, our goal is to have the user read the text beyond "No results" and a question seemed the best way to do it (versus bold text).

**Pull request merged**